web/client/components/widgets/view/WidgetsView.jsx

```diff
@@ -17,7 +17,7 @@ const ResponsiveReactGridLayout =
         widthProvider
     )(Responsive);
 const withGroupColor = require('../enhancers/withGroupColor');
-const DefaultWidget = withGroupColor(require('../widget/DefaultWidget'));
+const DefaultWidget = withGroupColor(require('../widget/DefaultWidget').default);
```

Comment:
no conversion of all require here ;) ?

Reply:
I didn't want to convert all the files at once :smile: or the PR can become easely not reviewable. but if you prefer I can convert also all the Widgets plugin (actually I converted only the Dashboard part)

Reply:
we can do it later on :)